### PR TITLE
4 get reviews

### PR DESCRIPTION
**Pull request by Vondry66**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+describe("GET by ID", () => {
+  it("200: should respond with a single review ", () => {
+    const REVIEW_ID = 3;
+    return request(app)
+      .get(`/api/reviews/${REVIEW_ID}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review).toEqual({
+          review_id: REVIEW_ID,
```
Comment (Augs0) [inline __tests__/app.test.js:46]: If we're testing for a specific article, we should be testing everything matches exactly e.g. we are not just checking the title is a string, but that the string is the same string as the article with id of 3

File: app.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
+
+app.use((err, req, res, next) => {
```
Comment (Augs0) [inline app.js:17]: The 500 error should always be the last in this chain as it's the last conclusion an error is likely to come to

**Review by Augs0 — CHANGES_REQUESTED** [review]
Please make the small changes mentioned in the comments below :)

File: app.js (hunk 2 above)
Comment (Vondry66) [reply app.js:17]: Thank you !

File: __tests__/app.test.js (hunk 1 above)
Comment (Vondry66) [reply __tests__/app.test.js:46]: Done that ! :)

**Pull request merged**